src/App.js

```diff
@@ -1,10 +1,23 @@
 import './App.css';
+import {useLocation} from 'react-router-dom'
+import { AllRoutes } from './Pages/AllRoutes';
+import { SideNav, Navbar } from './Components'
 
 function App() {
+
+  const {pathname} = useLocation();
   return (
-    <div className="App">
-      <h1>Hello World</h1>
-    </div>
+    <div>
+      {pathname!=="/"&&
+        <Navbar />
+      }
+      <div className='flex'>
+        {pathname!=="/"&&
+        <SideNav/>
+      }
```

Comment:
appreciation: Nice Maruthi, rendering navbar and sidebar if pathname doesn't match👏